doc/sphinx/source/community/detailed_release_procedure.rst

```diff
@@ -0,0 +1,190 @@
+.. _detailed-release-procedure:
+
+Release: recipes runs and comparison
+====================================
+
+The release procedure for ESMValTool is a fairly involved process (at the moment), so it
+is important to be very well organized and to have documented each procedural steps, so that
+the next release manager can follow said steps, and finalize the release without any delays.
+
+The workflow below assumes an ESMValCore release candidate, or a completed stable release, have been released
+and deployed on conda-forge and PyPI; it also assumes the release manager has access to accounts on DKRZ/Levante.
+
+Below is a list of steps that the release manager, together with the previous release manager, should go through before the actual release;
+these include testing the new code by running all available recipes in yhe `main` branch, and comparing the output against
+the previous release.
+
+Open an issue on GitHub
+-----------------------
+
+First, open an issue on GitHub where the testing workflow before the release is documented (see example https://github.com/ESMValGroup/ESMValTool/issues/2881).
+Name it something relevant like "Recipe testing and comparison for release 2.x.x", and populate the issue description with information
+about where the testing is taking place, what tools are used, and what versions, here are some suggestions:
+
+
+Where the work is done e.g. work done on DKRZ/Levante:
+
+- submit files in `/home/b/b382109/submit` (for v2.7.0)
+- output in `/scratch/b/b382109/esmvaltool_output` (for v2.7.0)
+
+We should document various utilities' versions so that the work can be reproduced in case there
+is an issue, or release work needs to be picked up mid-release by another release manager:
+
+- documenting `conda`/`mamba` versions:
+
+.. code-block:: bash
+
+  mamba --version
+
+- documenting `git` branch and its state:
+
+.. code-block:: bash
+
+  git status
+
+Furthermore, the runtime environment needs to be documented:
+
+.. code-block:: bash
+
+  mamba env create -n tool2xxTest -f environment.yml
+  conda activate tool2xxTest
+
+Make a copy of the environment file, and attach it in the release testing issue; to
+record the environment in a yaml file use e.g. `conda env export > ToolEnv270Test.txt`.
+
+Mention any special data movement one needs to perfom to run the tests e.g.:
+for v2.7.0 the release manager moved the autoassess-specific files to
+`/home/b/$USER/autoassess_files` on DKRZ/Levante.
```

Comment:
As stated in our [broken recipe policy](https://esmvaltool--3032.org.readthedocs.build/en/3032/community/broken_recipe_policy.html), all data should be publicly available and instructions to obtain it should be part of the recipe documentation. Why is this needed?

Reply:
ha! If only that were true :grin: I'd put this up for discussion - I need to leave this item at least for now so @remi-kazeroni can get his data on Levante (although I know Remi knows where it is, but just in case)

Reply:
IIRC we don't have redistribution rights on these files, so I don't think we should do that. There is a better solution (discussed in #2309) but I haven't managed to implement it yet. Perhaps the best option in the short term is to suppress/remove these broken recipes until we fix them?

Reply:
Thanks for your answer @alistairsellar. I will take a look at those recipes when I run them for the release. If these recipes can't be run because of non-publicly available files, I will follow your advice and mark these as broken.